drivers/modem/gsm_ppp.c

```diff
@@ -102,7 +102,7 @@ static struct gsm_modem {
 NET_BUF_POOL_DEFINE(gsm_recv_pool, GSM_RECV_MAX_BUF, GSM_RECV_BUF_SIZE,
```

Comment:
The commit message states the obvious (what is being changed, this does not need to be mentioned in the commit msg btw as that can seen from the commit itself) but not why the change is done. Could you elaborate in the commit message why this needs to be done?